modules/retail/pages/retail-uyuni-server-setup.adoc

```diff
@@ -0,0 +1,292 @@
+
+. In your web browser, enter the address for the {productname} {webui}.
+This address was provided after you completed setup.
+For more information, see xref:retail:retail-uyuni-server-setup.adoc#retail-server-setup-yast[].
+
+. Sign in to the {webui}, navigate to the menu:Create Organization[Organization Name] field, and enter your organization name.
+
+. In the menu:Create Organization[Desired Login] and menu:Create Organization[Desired Password] fields, enter your username and password.
+
+. Complete the ``Account Information`` fields, including an email for system notifications.
+
+. Click btn:[Create Organization] to finish creating your administration account.
+
+When you have completed the {productname} {webui} setup, you are taken to the menu:Home[Overview] page.
+
+
+
+== Add Software Channels
+
+Before you register {productname} branch servers and terminals to your {productname} Server, check that you have the {opensuse} product enabled, and the required channels are fully synchronized.
+
+The products you need for this procedure are:
+
+[[opensuse-channels-cli]]
+[cols="1,1,1,1,1", options="header"]
+.OpenSUSE Channels - CLI
+|===
+
+| OS Version
+| Base Channel
+| Client Channel
+| Updates Channel
+| Other Channels
+
+| {opensuse} Leap 15.2
+| opensuse_leap15_2
+| opensuse_leap15_2-uyuni-client
+| opensuse_leap15_2-updates
+| uyuni-proxy-stable-leap-152
+
+|===
+
+include::snippets/add_channels_cli.adoc[]
+
+
+
+== Check Synchronization Status
+
+include::snippets/check_sync_webui_uyuni.adoc[]
+
+include::snippets/check_sync_cli.adoc[]
+
+[IMPORTANT]
+====
+{opensuse} channels can be very large.
+Synchronization can sometimes take several hours.
+====
+
+
+
+== Trust GPG Keys on Clients
+
+include::snippets/trust_gpg.adoc[]
+
+
+
+
+
+// FIXME Starting from here, everything is preliminary
+// Feedback provided by Lukas
+== Create Activation Key for a Proxy (Branch Server) and the Retail Terminals
+
+For more information about creating activation keys, see xref:client-configuration:activation-keys.adoc[].
+
+The activation key for a proxy (branch server) must contain these child channels:
+
+* openSUSE Leap 15.2 Updates (x86_64)
+* Uyuni Client Tools for openSUSE Leap 15.2 (x86_64)
+* Uyuni Proxy Stable for openSUSE Leap 15.2 (x86_64)
+
+The activation key for retail terminals based on openSUSE Leap 15.2 must contain these child channels:
+
+* openSUSE Leap 15.2 Updates (x86_64)
+* Uyuni Client Tools for openSUSE Leap 15.2 (x86_64)
+
+
+
+
+== Register Clients
+
+To register your openSUSE clients, you need a bootstrap repository.
+By default, bootstrap repositories are automatically created, and regenerated daily for all synchronized products.
+You can manually create the bootstrap repository from the command prompt, using this command:
+
+----
+mgr-create-bootstrap-repo --with-custom-channels
+----
+
+For more information on registering your clients, see xref:client-configuration:registration-overview.adoc[].
+
+
+
+=== Proxy
```

Comment:
I feel like this needs a verb. Register? Also, probably should be "Branch Server".

Reply:
ok, but you actually must install an openSUSE proxy first.